ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -ex
+
+#sudo apt-get install -y rocblas rccl-dev rccl-rdma-sharp-plugins
+sudo apt install libstdc++-12-dev
+sudo apt remove -y rccl
```

Comment:
These installation scripts must be run as sudo, so the sudo can be removed from all lines here

Reply:
done